### PR TITLE
add key id for autograph

### PR DESCRIPTION
**Pull request by tiftran**

### CODE REVIEW

**Comment by tiftran** [comment]
we would need to update the version for `requests` for the circle ci job to pass....

File: normandy/settings.py [hunk 1]
```diff
@@ -371,6 +371,7 @@ def RAVEN_CONFIG(self):
     AUTOGRAPH_HAWK_SECRET_KEY = values.Value()
     AUTOGRAPH_SIGNATURE_MAX_AGE = values.IntegerValue(60 * 60 * 24 * 7)
     AUTOGRAPH_X5U_CACHE_BUST = values.Value(None)
+    AUTOGRAPH_KEYID = values.Value(None)
```
Comment (leplatrem) [inline normandy/settings.py:374]: nit: I think could safely add `"normandy"` as the default key id value

**Review by jaredlockhart — APPROVED** [review]
This all looks correct to me, thank you @tiftran .  Why do we have to update requests?  Your instincts are correct here, we should be changing as little as possible.

File: normandy/settings.py (hunk 1 above)
Comment (leplatrem) [reply normandy/settings.py:374]: makes sense 👌

**Pull request merged**